src/app/pages/IdxPage/index.jsx

```diff
@@ -9,7 +10,7 @@ const IdxPage = ({ pageData }) => {
 
   return (
     <main role="main">
-      <h1 id="content">IDX Page</h1>
+      <IndexHeading id="content">IDX Page</IndexHeading>
```

Comment:
I believe since PR https://github.com/bbc/simorgh/pull/6662 is merged in, we can use the title from the pageData now.

Reply:
Yep, I'm working on it atm.